### ISSUE TITLE
Volta 2.0.0 now offer Linux Arm64 prebuilt binaries

### ISSUE THREAD

**Issue by iNViTiON** [MyneTop/volta#2]
Maybe we can rest, for now. 🥰

Thank you for everything.

<https://github.com/volta-cli/volta/releases/tag/v2.0.0>

**Comment by FourLeafTec** [comment]
Yes, I've noticed that as well. However, this library is still being used in a few of my other personal workflows, so I'll keep the automatic build running for a while but won't be actively maintaining it. I'll stop it once I no longer rely on this repository or if the automatic build becomes inoperable.
Perhaps I should add a note to the README to indicate that this repository is no longer recommended for use.

**Comment by iNViTiON** [comment]
Agreed! I also having a bunch project rely on this release too.

By the way, GitHub released GitHub Actions Runner with ARM support last month. It might be worth a side-quest if you're interested. 😊

<https://github.blog/news-insights/product-news/arm64-on-github-actions-powering-faster-more-efficient-build-systems/>

**Comment by FourLeafTec** [comment]
Thanks. I'll consider using this for future binary pre-builds.
